librz/main/rz-find.c

```diff
@@ -190,6 +195,67 @@ static int rzfind_open_file(RzfindOptions *ro, const char *file, const ut8 *data
 		return 0;
 	}
 
+	if (ro->import || ro->symbol) {
+		RzBinFile *bf;
+		const RzList *symbols, *imports;
+		RzListIter *iter, *it;
+		RzBinSymbol *symbol;
+		RzBinImport *import;
+		RzBin *bin = rz_bin_new();
+		RzIO *rio = rz_io_new();
+		RzBinOptions opt = { 0 };
+
+		if (!bin || !rio) {
+			result = 1;
+			goto sym_end;
+		}
+
+		rz_io_bind(rio, &bin->iob);
+		rz_bin_options_init(&opt, 0, 0, 0, false, false);
+
+		bf = rz_bin_open(bin, file, &opt);
+		if (!bf) {
+			result = 1;
+			goto sym_end;
+		}
+
+		if (ro->import) {
+			imports = rz_bin_get_imports(bin);
+			rz_list_foreach (ro->keywords, iter, kw) {
+				rz_list_foreach (imports, it, import) {
+					if (!strcmp(import->name, kw)) {
+						printf("ordinal: %d %s\n", import->ordinal, kw);
+						break;
```

Comment:
I think we should remove this. In the rare (possible though, I think) case that there are multiple entries with the same import name.

Reply:
Sorry for the late response. Remove `break;` already.